backend/ponder.config.ts

```diff
@@ -269,6 +269,10 @@ const zkSyncNetworks: zkSyncNetworks = Object.keys(networks)
 
 export default createConfig({
   blocks: {
+    RelayPoolSnapshot: {
+      interval: 25,
+      network: vaultSnapshotNetworks,
+    },
```

Comment:
I think it is ok to start with 25, but we probably want to distinguish between some chains where the block frequency is different